tests/testthat/test-pmcmc.R

```diff
@@ -5,157 +5,189 @@ context("pmcmc")
 test_that("pmcmc runs without error", {
   data <- read.csv(sircovid_file("extdata/example.csv"),
                    stringsAsFactors = FALSE)
-  
+
   cmp <- readRDS("reference_pmcmc.rds")
 
   n_mcmc <- 10
   set.seed(1)
-  sircovid_model <- basic_model(gammas = list(E = 1/(2.5), 
-                                              asympt = 1/2.09, 
-                                              mild = 1/2.09, 
-                                              ILI = 1/4, 
-                                              hosp = 2, 
-                                              ICU = 2/5, 
+  sircovid_model <- basic_model(gammas = list(E = 1/(2.5),
+                                              asympt = 1/2.09,
+                                              mild = 1/2.09,
+                                              ILI = 1/4,
+                                              hosp = 2,
+                                              ICU = 2/5,
                                               rec = 2/5))
 
   X <- pmcmc(
-    data = data, 
+    data = data,
     n_mcmc = n_mcmc,
     sircovid_model = sircovid_model,
     pars_obs = list(phi_ICU = 0.95,
                     k_ICU = 2,
                     phi_death = 926 / 1019,
                     k_death = 2,
-                    exp_noise = 1e6)
+                    exp_noise = 1e6),
+    pars_to_sample = c(
+      'beta_start' = TRUE,
```

Comment:
Is there a test where either `beta_start` or `start_date` are set to `FALSE` here? I didn't follow how they would get turned off